MANIFEST.in

```diff
@@ -1,29 +1,17 @@
 include README.rst
 include CHANGES.rst
 
-include ez_setup.py
-include ah_bootstrap.py
-include setup_helpers.py
-include setup.cfg
-
-recursive-include *.pyx *.c *.pxd
-recursive-include docs *
 recursive-include licenses *
```

Comment:
We could include the license file by specifying it's name in `setup.cfg`, and then remove it here.  Though this requires `setuptools>=40.8`.  I think that's probably the better way to do it, as then it is also included in the wheel.  Recall MANIFEST.in` only affects the `sdist`, not the `bdist_wheel`.

Reply:
There's a `SUNPY_LICENSE.rst` file in there too, but that file isn't necessary now that asdf/extern/decorators.py has been removed.  I'll make this change.